core/trino-main/src/test/resources/hydra/ssl.cnf

```diff
@@ -0,0 +1,19 @@
+[ req ]
```

Comment:
Do we need this at all? Since we're accessing via `localhost`, we could use the existing `localhost` certificate